contracts/impls/centralized/WitnetRequestBoard.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.5;
+
+// Inherits:
+import "../../data/WitnetBoardDataWhitelists.sol";
+import "../../exports/WitnetRequestBoardInterface.sol";
+import "../../exports/WitnetBoard.sol";
+import "../../utils/Destructible.sol";
+import "../../utils/Upgradable.sol";
+
+// Uses:
+import "../../exports/WitnetRequest.sol";
+
+/**
+ * @title Witnet Requests Board mocked
+ * @notice Contract to bridge requests to Witnet for testing purposes.
+ * @dev This contract enables posting requests that Witnet bridges will insert into the Witnet network.
+ * The result of the requests will be posted back to this contract by the bridge nodes too.
+ * @author Witnet Foundation
+ */
+contract WitnetRequestBoard
+    is 
+        Destructible,
+        Upgradable, 
+        WitnetBoard, 
+        WitnetBoardDataWhitelists
+{
+    uint256 internal constant __ESTIMATED_REPORT_RESULT_GAS = 102496;
+
+    constructor() {
+        // sets instance as initialized:
+        __data().instance = address(this);
+    }
+
+
+    // ================================================================================================================
+    // --- Overrides 'Destructible' -----------------------------------------------------------------------------------
```

Comment:
I don't like this pattern of separating implementations using comments because it's very fragile. I prefer interfaces over inheritance, but not sure if it makes sense here.

Reply:
Just to make life of eventual code ammenders easier. I agree implementations blocks have to be preserved whenever inherited interfaces/contracts are changed, or new ones added.